### PR TITLE
Building portfolio

### PR DESCRIPTION
**Pull request by Najeeba-Qarqin**
This is a portfolio project created using HTML and CSS. 
Annotation 2024-05-30 194750

### CODE REVIEW

**Comment by Najeeba-Qarqin** [comment]
@BreshnaAndar786 please share your feedback.

**Comment by Najeeba-Qarqin** [comment]
@ahzamir I have finished the portfolio project.

**Comment by Najeeba-Qarqin** [comment]
@ahzamir Please review.

**Comment by Najeeba-Qarqin** [comment]
@ahzamir I'm waiting for your feedback.

File: index.html
Comment (ahzamir) [inline index.html:1]: Your HTML code is not clean and properly aligned. Please use 2 spaces for indentation instead of 0-1-3-4. Additionally, ensure that child elements are properly indented under their parent elements. Here's an example to help you:

**Incorrect:**
```html
<!DOCTYPE html>
<html>
    <head>
        <title>Page Title</title>
    </head>
    <body>
        <h1>This is a Heading</h1>
        <p>This is a paragraph.</p>
    </body>
</html>
```

**Correct:**
```html
<!DOCTYPE html>
<html>
  <head>
    <title>Page Title</title>
  </head>
  <body>
    <h1>This is a Heading</h1>
    <p>This is a paragraph.</p>
  </body>
</html>
```

**Rule:** Each child element should be indented 2 spaces under its parent element for better readability and maintainability.
**Hint**: Go to the VSCode bottom right corner, you can see "Spaces: 4" (or a similar indicator). Click on that, then click on "Indent Using Spaces" and select 2. You can also use the gray color column lines to help you know where your parent and child elements should be.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Najeeba-Qarqin 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Well done Najeeba you have made a great portfolio webpage with a great design, although your codes passes the freecodecamp test but still you need to clean some of your code in order to get the final approval for your project.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Personal Portfolio webpage and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌Your code is clean
✔️Your repo has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Najeeba-Qarqin** [comment]
@ahzamir I implemented the feedback now the codes are clean.👍

**Review by ahzamir — APPROVED** [review]
### Hi @Najeeba-Qarqin 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Odin project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**
